### PR TITLE
startup script for centos, with documentation.

### PR DESCRIPTION
**Pull request by joshrabinowitz**
Howdy, here's a pull request for bitcoind startup script for CentOS systems.

As discussed with ajweiss on IRC.

Let me know if you have feedback or questions.

### CODE REVIEW

**Comment by paveljanik** [comment]
Please add `-disablewallet` argument (maybe in the default example `sysconfig/bitcoind` file?). System-wide service should not have wallet enabled by default.

Please squash afterwards.

File: contrib/init/bitcoind.init [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        . /etc/sysconfig/bitcoind
+fi
+
+RETVAL=0
+
+prog=bitcoind
+lockfile=/var/lock/subsys/bitcoind
+
+# bitcoind defaults to /usr/sbin/bitcoind
+bitcoind=${BITCOIND-/usr/sbin/bitcoind}
```
Comment (luke-jr) [inline contrib/init/bitcoind.init]: bitcoind is installed to bin, not sbin

**Comment by luke-jr** [comment]
Probably would be better to use the [same variable names](https://github.com/bitcoin/bitcoin/blob/master/contrib/init/bitcoind.openrcconf) as the existing init script, unless there's some RedHat/CentOS policy on how the variables ought be named.

**Comment by joshrabinowitz** [comment]
@luke-jr , @paveljanik : I think I've reflected all the comments that have been input. 

There's been some discussion about the path to bitcoind-- I understand that it's currently installed by default into /usr/bin/bitcoind , but pavel would like it put into /usr/sbin/bitcoind. Please let me know which way I should go with this script.

**Comment by paveljanik** [comment]
After a few tests, I'd like to PR installing bitcoind into sbin instead of bin: https://github.com/paveljanik/bitcoin/commit/af092db6c81a6bad6208adaf5f645c8e3c5d6996

**Comment by luke-jr** [comment]
Currently bitcoind is installed into bin, and the init script should reflect that. When/if that changes, it can be updated here at that time. (FWIW, I think I disagree with installing bitcoind into sbin, but that is not a conversation for this PR)

**Comment by joshrabinowitz** [comment]
@luke-jr , @paveljanik : all sounds good to me. Let me know if there's additional feedback on this PR

**Comment by jgarzik** [comment]
ut ACK

**Comment by joshrabinowitz** [comment]
I just tested this on Centos 6.6 using this script:  
https://github.com/joshrabinowitz/bitcoin-centos-installer/blob/master/bitcoind-install
and it works: 
```
# service bitcoind stop
Stopping bitcoind:                                         [  OK  ]
# service bitcoind status
bitcoind is stopped
# service bitcoind start
Starting bitcoind: Bitcoin server starting
                                                           [  OK  ]
# service bitcoind status
bitcoind (pid 22883) is running...
```

**Pull request merged**